### PR TITLE
The Great Role Refluffing

### LINKED ISSUE #9: Clown/Mime/Musician Merger
### Maintainer Feature Request

**Feature description:** merge clown, mime and musician access levels into one single job: Entertainer, with 2 job slots.

**Lore justification (if applicable):** clowns mimes and musicians are all very small jobs and can be easily handled by 2 entertainers doing little roleplay bits together.

**Additional context:** If you want to join in go bug the captain to reassign you it's no big deal.

### LINKED ISSUE #14: Remove Martial Artist, Gladiator, Boxer
### Maintainer Feature Request

**Feature description:** all-access all boxing related areas, remove the job slots in question.

**Lore justification (if applicable):** NT wouldn't pay for this.

**Additional context:** just ask an HoP to change your ID.

### LINKED ISSUE #15: Merge Zookeeper into Botanist
### Maintainer Feature Request

**Feature description:** remove zookeeper job slot, give zookeeper access to botany.

**Lore justification (if applicable):** zookeeper is like botanist for animals.

**Additional context:** if it's done this way HoP can assign zookeeper separately if it ever actually matters.

### LINKED ISSUE #17: Temporarily Remove Librarian
### Maintainer Feature Request

**Feature description:** remove librarian job slot.

**Lore justification (if applicable):** n/a

**Additional context:** When the library system is implemented, we'll add it back, but it's a low-priority goal for us as a maintainer. There's just not enough to do as a librarian to justify it yet, though it pains me. HoP can just change role.

### LINKED ISSUE #19: Chaplain Back To Civvie
### Maintainer Feature Request

**Feature description:** make the chaplain a civilian role again.

**Lore justification (if applicable):** chaplain is not a scientist.

**Additional context:** This made sense when Epistemics was a thing, but we are putting it back to Research in our lore.

### LINKED ISSUE #20: Remove Prison Guard
### Maintainer Feature Request

**Feature description:** remove prison guard, this is a warden job.

**Lore justification (if applicable):** 

**Additional context:**

### LINKED ISSUE #21: Cut Service Worker
### Maintainer Feature Request

**Feature description:** its another Assistant+ role but one that doesn't make sense in our roleplay.

**Lore justification (if applicable):** 

**Additional context:** we MIGHT need to refluff passenger as assistant?

### LINKED ISSUE #22: Captain to Station Administrator
### Maintainer Feature Request

**Feature description:** rename Captain to Station Administrator.

**Lore justification (if applicable):** we're a got dang station we don't have a captain.

**Additional context:**

### LINKED ISSUE #23: Rebrand Lawyer
### Maintainer Feature Request

**Feature description:** lawyers are now Internal Affairs Agents, rename them and their items as such.

**Lore justification (if applicable):** NT internal affairs roots out corruption, audits and verifies paperwork, ensures employees are only being exploited the usual amount (to avoid lawsuits and nothing else).

**Additional context:** lawyer is also just a very narrow roleplay role that doesn't get to do much else than wait for people to go to court.

### LINKED ISSUE #24: Rebrand Mantis, Mystagogue
### Maintainer Feature Request

**Feature description:** Mantis becomes some kind of appropriate anomaly control job name; Mystagogue back to Research Director. Give the RD normal outfits if necessary, not sure how Mystagogue outfits look.

**Lore justification (if applicable):** NT is a normal science company, not a cult... yet.

**Additional context:** another Delta-V ism that won't stick around.

### LINKED ISSUE #25: Cut Corpsman
### Maintainer Feature Request

**Feature description:** remove the corpsman job slot.

**Lore justification (if applicable):**

**Additional context:** discourages medical crossplay.

### PR DESCRIPTION
**Pull request by revsys413**
# FIRST PR WOOO!!!

## About the PR

- Combined Musician, Clown, Mime into Entertainer
- Rolled clown and mime antag items into Entertainer
- Rolled Zookeeper into Botanist
- Put Chaplain back in Civilian
- Refluffed Psionic Mantis to Anomaly Control Specialist
- Refluffed Mystagogue back to Research Director
- Refluffed lawyer to IAA (Internal Affairs Agent)
- Refluffed Captain to Station Administrator
- Renamed syndie to traitor and nukie to mercenary
- Removed librarian temporarily
- Removed prison guard, service worker, corpsman, martial artist, gladiator, boxer

## Why / Balance

Resolves #9
Resolves #14
Resolves #15
Resolves #17
Resolves #19
Resolves #20
Resolves #21
Resolves #22
Resolves #23
Resolves #24
Resolves #25

## Technical details

- Removes access components from library and boxer doors
- Changes zoo access door components to hydro access components
- Streamlines mime, clown, musician access components to Theatre access
- Whitelists Entertainer for mime and clown antag items in the uplink
- Comments out some roles from the latejoin/character preference interface, shuffles around departments for some others
- Refluffs some role name and description strings

## Media

[i swear github can embed videos??](https://youtu.be/_EjpCIVRC7Q)
(kinda long, feel free to skip around)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

- tweak: Refluffed a lot of role related stuff.

### CODE REVIEW

**Comment by nearlyNonexistent** [comment]
Please [squash the commits.](https://phoenixnap.com/kb/git-squash#:~:text=Squashing%20commits%20in%20Git%20means,structure%20and%20remove%20redundant%20commits.)

**Comment by nearlyNonexistent** [comment]
Please [squash the commits.](https://phoenixnap.com/kb/git-squash#:~:text=Squashing%20commits%20in%20Git%20means,structure%20and%20remove%20redundant%20commits.)

**Comment by nearlyNonexistent** [comment]
testing if i got the githbu webhook working

**Comment by nearlyNonexistent** [comment]
Please [squash the commits.](https://phoenixnap.com/kb/git-squash#:~:text=Squashing%20commits%20in%20Git%20means,structure%20and%20remove%20redundant%20commits.)

File: .vscode/settings.json
Comment (nearlyNonexistent) [inline .vscode/settings.json:1]: Uninclude this file.

File: Resources/Locale/en-US/job/job-description.ftl [hunk 1]
```diff
@@ -28,10 +28,10 @@ job-description-hop = Handle access reassignment fairly using your ID console, m
...
 job-description-intern = Learn the basics of administering medicine, basic chemicals and using medical tools.
 job-description-janitor = Keep the station clean of any trash or slipping hazards, and help deal with rat infestations.
-job-description-lawyer = Ensure that every prisoner or criminal receives a fair judgment and trial if necessary.
+job-description-lawyer = Keep everything by the book and following SOP. Use your knowledge of corporate regulation to uphold Bureaucracy. Audit paperwork. Root out corrpution. Make the entire station groan when you show up.
 job-description-librarian = Manage the library, give out knowledge to any who seek it, and report on activities aboard the station.
 job-description-mime = Entertain the crew through non-vocal means, and engage with light rivalry with the clown.
-job-description-musician = Entertain the crew with your unique musical talent, and acquire new instruments to mess around with.
+job-description-musician = Entertain the crew with your wit, your silliness, your music, or your silence--but most importantly, your imagination.
```
Comment (nearlyNonexistent) [inline Resources/Locale/en-US/job/job-description.ftl]: Entertain the crew with performance art, music, or other talents -- but most importantly, your imagination.

File: Resources/Locale/en-US/nyanotrasen/job/job-names.ftl [hunk 2]
```diff
@@ -3,8 +3,8 @@ job-name-guard = Prison Guard
 job-name-mail-carrier = Courier
 job-name-martialartist =  Martial Artist
 job-name-prisoner = Prisoner
-job-name-mantis = Psionic Mantis
+job-name-mantis = Anomaly Control Specialist
```
Comment (nearlyNonexistent) [inline Resources/Locale/en-US/nyanotrasen/job/job-names.ftl]: We really have to figure out a shorter name for this...

File: Resources/Prototypes/DeltaV/Access/service.yml [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
+#  name: id-card-access-level-musician
 
 - type: accessLevel
   id: Reporter
   name: id-card-access-level-reporter
 
-- type: accessLevel
-  id: Zookeeper
-  name: id-card-access-level-zookeeper
+#- type: accessLevel
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/DeltaV/Access/service.yml]: I would be fine with leaving the Zookeeper access level separate and giving it to Botany.

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 4]
```diff
@@ -379,33 +379,33 @@
   parent: Airlock
   id: AirlockBoxerLocked
   suffix: Boxer, Locked
-  components:
-  - type: AccessReader
-    access: [["Boxer"]]
+  #components:
+  #- type: AccessReader
+  #  access: [["Maintenance"]] # Horizon - basically AA
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml]: Change the commented out line back to what it was, if it doesn't apply anymore.

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 5]
```diff
@@ -793,33 +793,33 @@
   parent: AirlockGlass
   id: AirlockBoxerGlassLocked
   suffix: Boxer, Locked
-  components:
-  - type: AccessReader
-    access: [["Boxer"]]
+  #components:
+  #- type: AccessReader
+  #  access: [["Maintenance"]] # Horizon - basically AA
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml]: Change the commented out line back to what it was, if it doesn't apply anymore.

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 6]
```diff
@@ -833,17 +833,17 @@
   parent: AirlockGlass
   id: AirlockLibraryGlassLocked
   suffix: Library, Locked
-  components:
-  - type: AccessReader
-    access: [["Library"]]
+  #components:
+  #- type: AccessReader
+  #  access: [["Maintenance"]] # Horizon - basically AA
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml]: Change the commented out line back to what it was, if it doesn't apply anymore.

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 7]
```diff
@@ -1108,33 +1108,33 @@
   parent: AirlockMaint
   id: AirlockMaintBoxerLocked
   suffix: Boxer, Locked
-  components:
-  - type: AccessReader
-    access: [["Boxer"]]
+  #components:
+  #- type: AccessReader
+  #  access: [["Maintenance"]] # Horizon - basically AA
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml]: Change the commented out line back to what it was, if it doesn't apply anymore.

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 8]
```diff
@@ -1148,17 +1148,17 @@
   parent: AirlockMaint
   id: AirlockMaintLibraryLocked
   suffix: Library, Locked
-  components:
-  - type: AccessReader
-    access: [["Library"]]
+  #components:
+  #- type: AccessReader
+  #  access: [["Maintenance"]] # Horizon - basically AA
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml]: Change the commented out line back to what it was, if it doesn't apply anymore.

File: Resources/Prototypes/DeltaV/Access/service.yml [hunk 9]
```diff
@@ -1,27 +1,29 @@
-- type: accessLevel
-  id: Boxer
-  name: id-card-access-level-boxer
+# Horizon - Remove basically all of these lmao
 
-- type: accessLevel
-  id: Clown
-  name: id-card-access-level-clown
+#- type: accessLevel
+#  id: Boxer
```
Comment (nearlyNonexistent) [inline Resources/Prototypes/DeltaV/Access/service.yml:4]: The order on these 3 seems to have changed and is bloating the changelog -- put the order back, if that's the case?

File: .vscode/settings.json [hunk 10]
```diff
@@ -1,4 +1,5 @@
 {
     "omnisharp.analyzeOpenDocumentsOnly": true,
-    "dotnet.defaultSolution": "SpaceStation14.sln"
+    "dotnet.defaultSolution": "SpaceStation14.sln",
```
Comment (nearlyNonexistent) [inline .vscode/settings.json]: Do not include changes to this file.

**Review by nearlyNonexistent — CHANGES_REQUESTED** [review]
Fix these and we can merge.

File: Resources/Locale/en-US/nyanotrasen/job/job-names.ftl (hunk 2 above)
Comment (nearlyNonexistent) [reply Resources/Locale/en-US/nyanotrasen/job/job-names.ftl]: Approved by host: Containment Specialist.

File: Resources/Prototypes/DeltaV/Access/service.yml (hunk 3 above)
Comment (revsys413) [reply Resources/Prototypes/DeltaV/Access/service.yml]: That would work, should I re-add the accesses to zoo doors then?

Comment (nearlyNonexistent) [reply Resources/Prototypes/DeltaV/Access/service.yml]: Yes.

File: Resources/Prototypes/DeltaV/Access/service.yml (hunk 9 above)
Comment (revsys413) [reply Resources/Prototypes/DeltaV/Access/service.yml:4]: Seems like the extra comment line I added made the changelog confused by shafting everything down -- should I put it at the end or something?

Comment (nearlyNonexistent) [reply Resources/Prototypes/DeltaV/Access/service.yml:4]: Eh, in that case leave it alone.

File: Resources/Maps/shoukou.yml
Comment (revsys413) [inline Resources/Maps/shoukou.yml:1]: I didn't change this file????????????? Huh?????????

**Comment by nearlyNonexistent** [comment]
Please squash the commits.

File: Resources/Maps/shoukou.yml
Comment (nearlyNonexistent) [inline Resources/Maps/shoukou.yml:1]: Don't include this change.

**Review by nearlyNonexistent — APPROVED** [review]

**Comment by nearlyNonexistent** [comment]
If I ever split anything into this many issues again, kill me.

**Pull request merged**